packages/sui-studio/src/components/style/index.js

```diff
@@ -19,8 +19,12 @@ export default class Style extends Component {
     this._linkElement = createLinkElement()
   }
 
+  componentWillUnmount () {
+    const lastIndex = document.styleSheets.length - 1
+    document.styleSheets[lastIndex].disabled = true
+  }
```

Comment:
Why? Do you think that href might change? As document.styleSheets is an array of CSSStyleSheet objects, which inherits from the StyleSheet object (see its [interface](https://drafts.csswg.org/cssom/#stylesheet)), we've a problem if the href might change!